docs/data-sources/external_dns.md

```diff
@@ -3,12 +3,13 @@
 page_title: "dcloud_external_dns Data Source - terraform-provider-dcloud"
 subcategory: ""
 description: |-
-  All the dns records currently in a given topology
+  All the External DNS rules in a given topology
 ---
 
 # dcloud_external_dns (Data Source)
 
-All the dns records currently in a given topology
+All the External DNS rules in a given topology
```

Comment:
The previous version of the docs seem to have been hand edited, they should only be updated by `go generate`.

This is replacing the hand editing by the generated docs.

The descriptions in the data source code should be updated to the correct wording, I'll open an issue for this as it's outside the scope of this change.